finetuning-bert/Fine-tuning-BERT.ipynb

```diff
@@ -2,8 +2,6 @@
  "cells": [
```

Comment:
<p>Need to also remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-Pytorch-fork/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>